app_wshwsim/sl_rf_driver.c

```diff
@@ -41,7 +44,17 @@ static uint8_t rf_mac_address[8];
 static int8_t rf_driver_id = (-1);
 static bool data_request_pending_flag = false;
 static uint16_t channel = 0;
+static bool rf_state = false;
+static int tx_time = 0;
+static bool cca_enabled = false;
```

Comment:
I checked this: although we can embed these 3 global variables inside `wsmac_ctxt`, but since they are directly used in the driver as `channel` is used and we don't want to share them among other files, I prefer to let them to be like this. We only need `rf_frame_cca_progress` inside `sl_wsrcp.c` to prevent RX when there is CCA or TX going on. Fortunately `rf_frame_cca_progress` is already in `wsmac_ctxt` :)